### PR TITLE
www-client/elinks: version bump

### PR DESCRIPTION
**Pull request by doughdemon**

### CODE REVIEW

File: www-client/elinks/elinks-0.13.5.ebuild [hunk 1]
```diff
@@ -3,23 +3,20 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python2_7 )
-PYTHON_REQ_USE="threads"
+PYTHON_COMPAT=( python3_{7,8,9} )
```
Comment (thesamesam) [inline www-client/elinks/elinks-0.13.5.ebuild:6]: Any chance of keeping 6? OK if not.

File: www-client/elinks/elinks-0.13.5.ebuild [hunk 2]
```diff
@@ -28,25 +25,28 @@ RDEPEND="
...
+			libressl? ( dev-libs/libressl:0= )
+		)
+		gnutls? ( net-libs/gnutls:= )
 	)
 	tre? ( dev-libs/tre )
 	X? ( x11-libs/libX11 x11-libs/libXt )
 	xml? ( >=dev-libs/expat-1.95.4 )
-	zlib? ( >=sys-libs/zlib-1.1.4 )"
+	zlib? ( >=sys-libs/zlib-1.1.4 )
+	zstd? ( app-arch/zstd )"
```
Comment (thesamesam) [inline www-client/elinks/elinks-0.13.5.ebuild:44]: :=

File: www-client/elinks/elinks-0.13.5.ebuild [hunk 3]
```diff
@@ -28,25 +25,28 @@ RDEPEND="
...
-		!libressl? ( dev-libs/openssl:0= )
-		libressl? ( dev-libs/libressl:0= )
+		!gnutls? (
+			!libressl? ( dev-libs/openssl:0= )
+			libressl? ( dev-libs/libressl:0= )
+		)
+		gnutls? ( net-libs/gnutls:= )
 	)
 	tre? ( dev-libs/tre )
 	X? ( x11-libs/libX11 x11-libs/libXt )
```
Comment (thesamesam) [inline www-client/elinks/elinks-0.13.5.ebuild:41]: Split into two lines because multiple deps

File: www-client/elinks/elinks-0.13.5.ebuild [hunk 4]
```diff
@@ -71,12 +71,13 @@ src_configure() {
 		$(use_with guile)
 		$(use_with idn)
 		$(use_with javascript spidermonkey)
-		$(use_with lua)
+		--with-luapkg=$(usev lua)
```
Comment (thesamesam) [inline www-client/elinks/elinks-0.13.5.ebuild:74]: $(use_with lua luapkg) may work?

**Comment by thesamesam** [comment]
ping

**Comment by doughdemon** [comment]
thanks for the comments, and sorry for the delay.

**Pull request closed without merging**